src/helpers/CatalogHelper.ts

```diff
@@ -67,27 +67,27 @@ export default class CatalogHelper {
     }
 
     public measure(name: string): string {
-        return get<CatalogHelper, string>(this, ["measures", name, "identifier"]);
+        return get(this, ["measures", name, "identifier"]);
```

Comment:
just wondering what is the motivation behind this changes?

Reply:
We were unable to properly type these getters after upgrading TypeScript. The second parameter (key) should be one of the keys of `this`, but it is an array of strings in our case. After several discussions, we were unable to find solution.